scripts/travis/upload-to-quay.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
Did you see @jpkrohling's comment? https://github.com/jaegertracing/jaeger/pull/2325/files#r509103626

I think you may not need this script; and should just need to modify the aforementioned script once #2325 is merged.

Reply:
no worries. I'll add my changes as soon as #2325 is merged